widget.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "File list",
+  "package": "com.fliplet.file-list",
+  "version": "1.0.0",
+  "icon": "img/icon.png",
+  "tags": ["type:component", "category:general"],
+  "provider_only": false,
+  "references": [
+    "content:richContent",
+    "dataSource:dataSource"
+  ],
+  "settings": {
+    "asyncRendering": true,
+    "interfaceWidgetPackage": "com.fliplet.helper-configuration",
+    "hideThemeSettings": true
+  },
+  "html_tag": "div",
+  "interface": {
+    "dependencies": [
+      "fliplet-core",
+      "fliplet-studio-ui",
+      "fliplet-datasources"
+    ],
+    "assets": [
+      "js/interface.js"
+    ]
+  },
+  "build": {
+    "dependencies": [
+      "jquery",
+      "bootstrap",
+      "fliplet-core",
+      "fliplet-helper",
+      "fliplet-datasources",
+      "photoswipe"
+    ],
+    "assets": [
+      "css/build.css",
+      "js/build.js"
+    ],
+    "appAssets": []
```

Comment:
You can omit this if you're not going to include any assets

```suggestion
```

Reply:
Done